File: ProjectTest/OneLang/CSharp/Parsers/Common/NodeManager.cs
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System;
using Parsers.Common;

namespace Parsers.Common
{
    public class NodeManager {
        public List<object> nodes;
        public Reader reader;
        
        public NodeManager(Reader reader)
        {
            this.reader = reader;
            this.nodes = new List<object>();
        }
        
        public void addNode(object node, int start) {
            this.nodes.push(node);
        }
    }
}

File: ProjectTest/OneLang/Java/src/main/java/OneLang/Parsers/Common/NodeManager/NodeManager.java
import java.util.List;
import java.util.ArrayList;

public class NodeManager {
    public List<Object> nodes;
    public Reader reader;
    
    public NodeManager(Reader reader)
    {
        this.reader = reader;
        this.nodes = new ArrayList<Object>();
    }
    
    public void addNode(Object node, Integer start) {
        this.nodes.add(node);
    }
}